commit ebeba1b01641fa6ad13813cc899c587554cc89cb (refs/changes/05/2005/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-05-25T16:02:49-07:00 (2 years, 4 months ago)
    
    joyent/node-sshpk#29 tests fail with openssl 1.1.0e because of extra spaces in DN output

diff --git a/test/openssl-cmd.js b/test/openssl-cmd.js
index ab27c93..a62d24f 100644
--- a/test/openssl-cmd.js
+++ b/test/openssl-cmd.js
@@ -222,10 +222,12 @@ function genTests() {
 			var output = Buffer.concat(bufs).toString();
 
 			var m = output.match(/Issuer: (.*)$/m);
-			t.strictEqual(m[1], 'CN=' + algo + ', C=US');
+			t.strictEqual(m[1].replace(/ = /g, '='),
+			    'CN=' + algo + ', C=US');
 
 			m = output.match(/Subject: (.*)$/m);
-			t.strictEqual(m[1], 'CN=' + algo + ', C=US');
+			t.strictEqual(m[1].replace(/ = /g, '='),
+			    'CN=' + algo + ', C=US');
 
 			var re = /DNS:([^, \n]+)([, ]+|$)/gm;
 			m = re.exec(output);
